src/reference/asciidoc/jdbc.adoc

```diff
@@ -365,6 +365,9 @@ If your database is not listed, you can easily extend the `AbstractChannelMessag
 
 Since _version 4.0_, the `MESSAGE_SEQUENCE` column has been added to the table to ensure first-in-first-out (FIFO) queueing even when messages are stored in the same millisecond.
 
+Since_version 5.0, by overloading `MessageGroupPreparedStatementSetter` class you can provide custom implementation to 
+ `PreparedStatementSetter` to support different columns or table structure.
```

Comment:
Well, it is somehow wrong to say `PreparedStatementSetter` because the `MessageGroupPreparedStatementSetter` doesn't implement it, really. 

> Starting with _version 5.0_, the `JdbcChannelMessageStore` can be supplied with the custom `MessageGroupPreparedStatementSetter`  to support different columns or table structure or data types.
For example, instead of default message serialization to the `byte[]`, we can store its structure as JSON string.

Reply:
Yes. But the functionality is almost similar.

Reply:
Right, you say that in its JavaDocs, but that’s fully wrong to mislead target developer to say him/her to implement wrong contruct. That really might be also good to add simple sample here in Doc. For that original idea for just simple storing message in varchar column, leaving the rest in the same structure.